### PR TITLE
Docker compose configparser

### PR DESCRIPTION
**Pull request by orviz**

### CODE REVIEW

File: src/eu/indigo/compose/parser/ConfigParser.groovy [hunk 1]
```diff
@@ -11,107 +11,110 @@ class ConfigParser implements Serializable {
...
-    static def getServiceClass(def name) {
-        // TODO: Refactor this
-        switch(name) {
-            case "Postgres":
-                return Postgres
-                break
-            case "Postgis":
-                return Postgis
+    
+    static Map getDefaultValue(String setting) {
```
Comment (orviz) [inline src/eu/indigo/compose/parser/ConfigParser.groovy:46]: Default values should be put at the top in a closure

File: src/eu/indigo/compose/parser/ConfigParser.groovy [hunk 2]
```diff
@@ -11,107 +11,110 @@ class ConfigParser implements Serializable {
 
     private static String LATEST = 'latest'
     private static Integer DEFAULT_TIMEOUT = 600   // 600 seconds
```
Comment (orviz) [inline src/eu/indigo/compose/parser/ConfigParser.groovy:13]: These values must be merged into getDefaultValues

File: src/eu/indigo/compose/parser/ConfigParser.groovy [hunk 3]
```diff
@@ -11,107 +11,110 @@ class ConfigParser implements Serializable {
...
-
-        // parse the necessary services
-        projectConfiguration.services   = parseServices(yaml.services)
-
-        // load the dockefile
-        projectConfiguration.dockerfile = parseDockerfile(yaml.config)
-
-        // load the project name
+        projectConfiguration.environment = parseEnvironment(yaml.environment)
         projectConfiguration.projectName = parseProjectName(yaml.config)
```
Comment (orviz) [inline src/eu/indigo/compose/parser/ConfigParser.groovy:28]: @samuelbernardolip  do we need these? i.e. `buildNumber`, `environment`, `projectName`

Comment (samuelbernardolip) [reply src/eu/indigo/compose/parser/ConfigParser.groovy:28]: No, that is code from WoloxCI.

**Pull request merged**